crates/release_plz_core/src/pr.rs

```diff
@@ -64,7 +73,21 @@ fn pr_title(
             .all(|(_, update)| &update.version == first_version)
     };
 
-    if updates.len() == 1 && project_contains_multiple_pub_packages {
+    if let Some(title_template) = title_template {
+        let package_name = if updates.len() == 1 {
+            &updates[0].0.name
+        } else {
+            ""
+        };
+        let first_version = first_version.to_string().clone();
+        let version = if are_all_versions_equal() {
+            first_version.as_str()
+        } else {
+            ""
+        };
```

Comment:
sorry, I made a mistake, we shouldn't set the field if not present.
I.e. we shouldn't set empty string.

Reply:
should we let them unset? what would be the behavior in this case? the literal `{{ version }}` string appear in the message? or should we error?

Reply:
Failing seems the sanest way of handling things, as I think this feature will be used in very specific cases (looking at the original request). I added a commit to implement the change, let me know what you think

Reply:
Imo we should not set the fields. But i haven't looked into your changes yet.